components/footer/src/stories.tsx

```diff
@@ -46,6 +46,32 @@ WithCopyright.args = {
   },
 };
 
+export const WithCustomLicense = Template.bind({});
+
+WithCustomLicense.args = {
+  licence: (
+    <span>
+      All content is available under the{' '}
+      <Footer.Link href="https://creativecommons.org/licenses/by/4.0/" rel="license">
+        Creative Commons Attribution 4.0 International Licence
+      </Footer.Link>
+      , except where otherwise stated
+    </span>
+  ),
+};
+
+export const WithNoLicense = Template.bind({});
+
+WithNoLicense.args = {
+  meta: (
```

Comment:
do we need a custom meta for this story? Should it be a separate story?

Reply:
Sure - I've removed it :)

Just thought it looked a bit weird with nothing in the footer at all - but probably makes the story simpler to understand.

![image](https://user-images.githubusercontent.com/4953590/166155278-e1073f6f-b555-4489-9099-63855f93b731.png)